src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -59,35 +52,29 @@ public function showAction(Request $request)
 
     /**
      * @param Request $request
-     * @param string  $namespace
      *
      * @return Response
      */
-    public function updateAction(Request $request, $namespace)
+    public function updateAction(Request $request)
     {
-        $this->isGrantedOr403($namespace);
+        $schemaAlias = $request->get('schema');
 
-        $manager = $this->getSettingsManager();
+        $this->isGrantedOr403($schemaAlias);
```

Comment:
I think `name` a good name.  Not sure if it's descriptive enough though, since you can easily mix it with thinking you are loading by `namespace`, which you are obviously not doing.